Core/Domibus-MSH/src/main/resources/config/domibus-default.properties

```diff
@@ -111,6 +111,9 @@ domibus.plugin.account.unlock.cron=0 0/1 * * * ?
 
 # ---------------------------------- Payload -----------------------------------
 
+# Sets businessContentAttachment enabled in payload
+domibus.payload.business.content.attachment.enabled=true
```

Comment:
The default value should be `false` so that we don't change the existing behavior. In case the user wants to use this feature, the property value must be set to `true` manually.

Reply:
I have now set it to be false by default.